src/components/buttons/primary_button.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+
+export default function PrimaryButton({
+  clickFunction,
+  disabled,
+  width,
+  height,
+  buttonText,
+}) {
+  return (
+    <div
+      onClick={!disabled ? clickFunction : null}
+      style={{ width: width, height: height }}
+      className={
+        (disabled
+          ? "bg-[#4956F4]/70 hover:cursor-default"
+          : "hover:cursor-pointer hover:bg-[#5e6bfe]") +
+        ` transition-all duration-100 rounded-2xl flex justify-center text-white items-center font-medium bg-[#4956F4]`
+      }
+    >
+      {buttonText}
+    </div>
+  )
+}
+
+// Copy directly:
```

Comment:
why is this code commented out?

Reply:
Just having this reusable snippet to directly copy and paste wherever needed, will remove at the end